playbooks/roles/deploy-osh/templates/keystone.yaml.j2

```diff
@@ -1,2 +1,45 @@
 ---
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+{% if developer_mode %}
+conf:
+  apache2:
+    binary: apache2ctl
+    start_parameters: -DFOREGROUND -k start
+    site_dir: /etc/apache2/vhosts.d
+    conf_dir: /etc/apache2/conf.d
+    a2enmod:
+      - version
+    security: |
+      <Directory "/var/www">
+      Options Indexes FollowSymLinks
+      AllowOverride All
+      <IfModule !mod_access_compat.c>
+        Require all granted
+      </IfModule>
+      <IfModule mod_access_compat.c>
+        Order allow,deny
+        Allow from all
+      </IfModule>
+      </Directory>
+pod:
+  security_context:
+    keystone:
+      pod:
+        runAsUser: 0
+      container:
+        keystone_api:
+          readOnlyRootFilesystem: false
+images:
+  tags:
+    bootstrap: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_osh_image_version }}"
```

Comment:
In this patch, we will only focus on keystone, so I expect we should not update heat image version, even in developer mode. Could we create a new var for overriding the keystone images, and we could slowly phase all the overrides to use that image? 

Would that work with keystone <rocky> and the other images staying the same?

Reply:
well, that image is part of the keystone chart so it makes sense to bind them together no? After all, if we dont do this, we would get a keystone deployment that contains suse images + ubuntu images mixed in there.

I did understand that as part of the overrides we want to make the whole chart work under suse.

Reply:
ok so you mean you've tested only with keystone + heat images together?

Reply:
In this case, we should, when developer mode, still point to a different value than ocata (present in the var suse_osh_image_version).

I am not really fond of splitting this into 2 PRs, as they are just one single thing. But I am not in favor of updating suse_osh_image_version across the board for non-keystone.

Reply:
> ok so you mean you've tested only with keystone + heat images together?

yep, only tested with all suse images, no idea if it will work with a mix

> n this case, we should, when developer mode, still point to a different value than ocata (present in the var suse_osh_image_version).

Shouldnt that be done via the change in https://github.com/SUSE-Cloud/socok8s/pull/188 so we cna point to the new var across the board?

I can just use an extra var here `suse_osh_image_tag_kesytone` but it seems a bit superfluous to do it override by override where we could do it as a whole and have it synced on all overrides